### PR TITLE
Load seafile config from database

### PR DESCRIPTION
**Pull request by cuihaikuo**

### CODE REVIEW

File: server/seafile-session.c [hunk 1]
```diff
@@ -421,3 +431,24 @@ schedule_create_system_default_repo (SeafileSession *session)
...
+
+int
+create_config_table (SeafileSession *session)
+{
+    char *sql;
+    int db_type = seaf_db_type(session->db);
+
+    if (db_type == SEAF_DB_TYPE_SQLITE ||
+        db_type == SEAF_DB_TYPE_MYSQL)
+        sql = "CREATE TABLE IF NOT EXISTS SeafileConf "
```
Comment (shoeper) [inline server/seafile-session.c]: Isn't this usually part of the install / upgrade scripts?

Comment (cuihaikuo) [inline server/seafile-session.c]: Yes but not mandatory, it's ok here

Comment (shoeper) [inline server/seafile-session.c]: What if the DB user does not have permissions to create tables?

Comment (cuihaikuo) [inline server/seafile-session.c]: Actually there are many "CREATE TABLE" sqls in seafile-server, you can't run the server without  "CREATE" permission

**Comment by shoeper** [comment]
Ok, thanks for the clarification. Than I don't have any further objections.


Am 8. Juni 2017 07:36:12 schrieb cuihaikuo <notifications@github.com>:

**Pull request closed without merging**